hivemind/p2p/p2p_daemon.py

```diff
@@ -68,10 +66,12 @@ def __init__(self):
     @classmethod
     async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager: bool = True,
                      dht_mode: str = 'dht_server', force_reachability: Optional[str] = None,
-                     nat_port_map: bool = True, auto_nat: bool = True, bootstrap: bool = False,
```

Comment:
`bootstrap: bool` parameter does not make any sense: it is clear that the user wants to enable bootstrapping if either `bootstrap_peers` or `use_ipfs=True` is specified (and does not want to do it otherwise).